src/app/api/v1/realtime/collections.py

```diff
@@ -222,7 +223,7 @@ def score_weights_param(
 
 @router.get(
     "/collections/{noderef_id}/stats/score",
-    response_model=dict,
+    response_model=ScoreOutput,
```

Comment:
Can FastAPI actually get this from the return type annotation? I.e.
Would 

```python
@router.get( "/collections/{noderef_id}/stats/score", status_code=...)
async def score(...) -> ScoreOutput:
    ...
```
be equivalent?

Reply:
Great idea, I found this note: https://fastapi.tiangolo.com/advanced/response-directly/#notes
So basically, no. :/

Reply:
unfortunate :*(